src/Microsoft.DotNet.Build.Tasks/GenerateTestExecutionScripts.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class GenerateTestExecutionScripts : Task
+    {
+        [Required]
+        public string[] TestCommands { get; set; }
+
+        [Required]
+        public string[] TestDependencies { get; set; }
+
+        [Required]
+        public string RunnerScriptWindowsTemplate { get; set; }
+
+        // Not used yet.
+        [Required]
+        public string RunnerScriptUnixTemplate { get; set; }
+
+        [Required]
+        public string OutputFolder { get; set; }
+
+        public override bool Execute()
+        {
+            Log.LogMessage("Test Command lines = {0}", string.Join("\n", TestCommands));
+            WriteWindowsExecutionScript(OutputFolder);
+            return true;
+        }
+
+        private void WriteWindowsExecutionScript(string outputPath)
+        {
+            if (!File.Exists(RunnerScriptWindowsTemplate))
+            {
+                throw new FileNotFoundException($"Runner script template {RunnerScriptWindowsTemplate} was not found.");
+            }
+
+            string windowsExecutionTemplate = File.ReadAllText(RunnerScriptWindowsTemplate);
+            Directory.CreateDirectory(outputPath);
+
+            // Build up the copy commands... 
+            StringBuilder copyCommands = new StringBuilder();
+            copyCommands.AppendLine($"IF NOT EXIST  %EXECUTION_DIR% ( md %EXECUTION_DIR% )");
+            copyCommands.AppendLine($"IF NOT EXIST  %EXECUTION_DIR%\\core_root ( md %EXECUTION_DIR%\\core_root )");
```

Comment:
Do we still need the core_root directory? I don't see anything being copied to it.